modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,954 @@
+    },
+
+    ensembl_analysis_script => catdir( $self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts' ),
+    remove_duplicates_script_path => catfile( $self->o('ensembl_analysis_script'), 'find_and_remove_duplicates.pl' ),
+    remove_small_orf_script => catfile( $self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl' ),
+
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff'    => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi',    # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+
+########################
+# db info
+########################
+
+    'cdna_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_cdna_' . $self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_genblast_' . $self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'ig_tr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_igtr_'.$self->o('release_number'),
+      -host   => $self->o('ig_tr_db_server'),
+      -port   => $self->o('ig_tr_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'best_targeted_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_bt_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
```

Comment:
Could this be changed to best_targeted_db instead of genewise_db? This is to avoid confusion